crates/turborepo-lib/src/commands/logout.rs

```diff
@@ -0,0 +1,23 @@
+use anyhow::Result;
+use log::error;
+
+use crate::{
+    config::{default_user_config_path, UserConfig},
+    ui::{GREY, UI},
+};
+
+pub fn logout(ui: UI) -> Result<()> {
+    let mut config = UserConfig::load(
+        &default_user_config_path()
+            .ok_or_else(|| anyhow::anyhow!("Unable to find default config path"))?,
+        None,
+    )?;
+
+    if let Err(err) = config.set_token(None) {
+        error!("could not logout. Something went wrong: {}", err);
+        return Err(err);
+    }
+
+    ui.info(">>> Logged out", Some(&GREY))?;
```

Comment:
Would prefer using `log::info!` for this

I think we can delete the `UI` struct if we used another color library

Reply:
Turborepo has logs which respect verbosity level and then UI print statements which get written regardless, for UI info/error/warn/output just map to default styles applied to the string.